github_handler.py

```
@@ -0,0 +1,358 @@
+        async with aiohttp.ClientSession() as session:
+            async with session.get(url, headers=headers) as response:
+                if response.status != 200:
+                    logger.warning(f"GitHub API returned status {response.status} for languages of {owner}/{repo}")
+                    return None
+                
+                data = await response.json()
+                logger.info(f"Successfully fetched languages for {owner}/{repo}")
+                return data
+                
+    except Exception as e:
+        logger.error(f"Error fetching languages for {owner}/{repo}: {str(e)}", exc_info=True)
```

Ruff (0.14.3)

Use explicit conversion flag

Replace with conversion flag

(RUF010)